common/v2/features/SendAssets/components/TransactionReceipt.tsx

```diff
@@ -37,15 +38,19 @@ export default function TransactionReceipt({ txReceipt, txConfig }: IStepCompone
     );
     if (blockNumber === 0 && txReceipt.hash) {
       const blockNumInterval = setInterval(() => {
-        getTransactionReceiptFromHash(txReceipt.hash, provider).then(transactionReceipt => {
-          if (transactionReceipt) {
-            const transactionStatus = transactionReceipt.status === 1 ? true : false;
+        getTransactionReceiptFromHash(txReceipt.hash, provider).then(transactionOutcome => {
+          if (transactionOutcome) {
+            const transactionStatus = transactionOutcome.status === 1 ? true : false;
             setTxStatus(prevState => ({
               status: transactionStatus || prevState.status,
               known: transactionStatus !== undefined ? true : false
```

Comment:
naming: prefer the question form for booleans e.g `isKnown`. But here I wouldn't know what isKnown.
Since it's value is always the same as `transactionStatus` I doubt we need this extra state value.

Reply:
done.